app/controllers/chatfuel_controller.rb

```diff
@@ -7,14 +7,83 @@ def show
     if tc
       text = Text::Renderer.new(text_component: tc).render(:main_part)
 
-      @response = {
-        messages: [
-          { text: text.first(640)} # Messages for Facebook Messenger can only be 640 characters long. Source: https://developers.facebook.com/docs/messenger-platform/send-api-reference#request
-        ]
-      }
-      render json: @response
+      qa = tc.question_answers
+
+      #Test API Request: http://localhost:3000/chatfuel/1
+      if qa.exists?(1)
```

Comment:
"Returns true if a record exists in the table that matches the id or conditions given, or false otherwise."
-> So qa.exists?(1) will return true if a QuestionAnswer with the id 1 exists.

I think i see what you mean, the first QuestionAnswer record for a certain topic doesn't need to have the id 1. 

If used if qa.first instead, would that be the right implementation?